inductiva/tasks/task.py

```diff
@@ -155,3 +156,27 @@ class _PathParams(TypedDict):
 
     def _get_path_params(self) -> _PathParams:
```

Comment:
Can you comment on this method?
I know you didn't write it, but since it shows up here, I wonder why this is called _get_path_params and then we return a self.id?